torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1106,8 +1118,11 @@ struct GraphFuser {
         if (!canFuseWithConcat(input, fused_cat)) {
           continue;
         }
-        any_fused = true;
         auto maybe_group = tryFuse(fused_cat, input);
+        if( !maybe_group ) {
+          continue;
+        }
+        any_fused = true;
         AT_ASSERT(maybe_group && maybe_group == fused_cat);
```

Comment:
The first part of this assert is meaningless now

Reply:
Ok. Revision is done in the latest commit. Place a limit check in canFuseWithConcat() to restore the original code sequence around tryFuse() call in fuseConcats().